.github/CONTRIBUTING.md

```diff
@@ -12,20 +12,20 @@ If you are submitting a bug report, please include the following:
 
 * When submitting *bug reports*, *paste your schema example* whenever possible. This will save us from having to ask you for it later.
 * When submitting *Pull Requests (PRs)*, you should request to merge your changes into our `develop` branch.
-* Our approach to testing is described in [spec/README.md](/spec/README.md).
+* Our approach to testing is described in [docs/TESTING.md](/docs/TESTING.md).
 * PRs without specs will not be merged anytime soon!
 
 ## Releasing new versions
 
 * Create a new entry in the [CHANGELOG](/CHANGELOG) file,
-* Re-build and re-release the online demo, see [`website-jsf` gh-pages branch](https://github.com/json-schema-faker/website-jsf/tree/gh-pages).
+* Re-build and re-release the online demo, see [`json-schema-faker` gh-pages branch](https://github.com/json-schema-faker/json-schema-faker/tree/gh-pages).
 
 ## Development tasks
 
 * `npm run dev` &mdash; Run the tests and watch (preferred during development)
-* `npm run dev:lint` &mdash; Run eslint on all sources
-* `npm run dev:unit` &mdash; Run `jasmine-node` on unit tests
-* `npm run dev:schema` &mdash; Run `jasmine-node` on schema tests
+* `npm run lint` &mdash; Run eslint on all sources
+* `npm run test:unit` &mdash; Run unit tests
+* `npm run test:schema` &mdash; Run schema tests
```

Comment:
I would like to update the other npm scripts here, but idk what they should be.

Reply:
This is fine, I'll update those on time once I remove those we no longer need.